rotkehlchen/api/v1/types.py

```diff
@@ -11,3 +13,9 @@ class EvmTransactionDecodingApiData(TypedDict):
 class EvmPendingTransactionDecodingApiData(TypedDict):
     evm_chain: SUPPORTED_CHAIN_IDS
     addresses: Optional[list[ChecksumEvmAddress]]
+
+
+@dataclass(init=True, repr=True, eq=True, order=False, unsafe_hash=False, frozen=True)
+class IncludeExcludeFilterData:
+    values: list[HistoryBaseEntryType]
+    behaviour: Literal['IN', 'NOT IN'] = 'IN'
```

Comment:
Can you rename this here as `operator`? For the API leave it behaviour, but when translating to `IncludeExcludeFilterData` use this as it's the same we use in fitlersand it's also shorter.